### DISCUSSION TITLE
Colors in the price graphs

### DISCUSSION THREAD

**Discussion by LukasDoesDev** [miguelmota/cointop#196] [Ideas]
Could we get colors in the price graphs? Red indicating price down and green indicating price up?

**Comment by lyricnz** [comment]
The issue is that each horizontal interval represents a more-or-less arbitrary time period (scaled to current range, and how wide your screen is).  Is it really useful to know whether the "dot" is higher or lower than the previous?  Also, given this is a terminal application, there are actually two dots per "character", and we could only color characters, not dots

![Screen Shot 2021-10-03 at 10 59 17 am](https://user-images.githubusercontent.com/122371/135734803-423beec0-61f0-4964-a036-f07974205b37.png)
